backend/tracim_backend/lib/translate/services/systran.py

```diff
@@ -0,0 +1,140 @@
+from http import HTTPStatus
+import mimetypes
+from typing import BinaryIO
+from typing import List
+
+import requests
+
+from tracim_backend.lib.translate.translator import ExternalTranslator
+from tracim_backend.lib.translate.translator import TranslateService
+from tracim_backend.lib.translate.translator import TranslationFailed
+from tracim_backend.lib.translate.translator import TranslationLanguagePair
+from tracim_backend.lib.translate.translator import TranslationMimetypePair
+
+FILE_TRANSLATION_ENDPOINT = "/translation/file/translate"
+SUPPORTED_FORMAT_ENDPOINT = "/translation/supportedFormats"
+SUPPORTED_LANGUAGES_ENDPOINT = "/translation/supportedLanguages"
+
+
+class SystranFormat:
+    def __init__(self, name: str, mimetype_pair: TranslationLanguagePair):
+        self.name = name
+        self.mimetype_pair = mimetype_pair
+
+    def __repr__(self):
+        return "<Systranformat(name=%s, translation_pair=%s)>" % (
+            repr(self.name),
+            repr(self.mimetype_pair),
+        )
+
+
+class SystranTranslateService(TranslateService):
+    """
+    Sample example of translation service:
+    Do uppercase the text/plain file given.
+    """
```

Comment:
Is this docstring accurate? The class seems   like it's doing real work.

Reply:
no anymore effectively. i had to remove this.